capa/main.py

```diff
@@ -978,6 +978,41 @@ def handle_common_args(args):
         args.signatures = sigs_path
 
 
+def last_resort_exception_handler(args, exctype=None, value=None, traceback=None):
+    """
+    custom exception handler to replace the default sys.excepthook,
+    prints traceback in debug mode, otherwise prints helpful message
+
+    using "nonlocal" keyword and closure code construct enables
+    last_resort_exception_handler args to first be initialized and
+    then be updated if the function needs to handle an exception
+
+    args:
+      args (argparse.Namespace): parsed command-line arguments
+      exctype (Exception): exception class
+      value (str): exception instance
+      traceback (TracebackType): a traceback object
```

Comment:
alternatively/additionally set the types in the function so mypy can do type checking